addons/website/views/snippets/s_image_comparison.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="s_image_comparison" name="Image Comparison">
+        <section class="s_image_comparison pt24 pb24 o_not_editable" contenteditable="false">
+            <div class="o_container_small">
+                <div class="col-12">
```

Comment:
container > col is not a valid bootstrap structure... if we don't want to ability to resize the column, you don't even need it at all.